### ISSUE TITLE
qgpgme fails to build (failing test), blocking nixos-unstable

### ISSUE THREAD

**Issue by FRidh** [NixOS/nixpkgs#109009]
**Describe the bug**
https://hydra.nixos.org/build/134668103

Introduced by gnupg update, updating gpgme did not fix the issue.
https://github.com/NixOS/nixpkgs/pull/108423#issuecomment-757441419

**To Reproduce**
1. ...
2. ...
3. ...

**Expected behavior**

**Screenshots**

**Additional context**

**Notify maintainers**

@primeos

**Metadata**
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
# a list of nixos modules affected by the problem
module:
```

**Comment by primeos** [comment]
I gave it a quick look. The test failure of `libsForQt5.qgpgme` is indeed a regression from the most recent GnuPG update: 374e3033d18897d753e5d5f1e5c5849489dcca12.

Not sure though why it fails though (unfortunately I don't really have time to investigate this atm) and I didn't find any upstream issues or other reports of such failures so far (given that GnuPG 2.2.26 was released 2020-12-21 this regression might not affect (some/most) other Linux distributions).
